src/coreclr/inc/readytorun.h

```diff
@@ -397,7 +397,7 @@ struct READYTORUN_EXCEPTION_CLAUSE
 enum ReadyToRunRuntimeConstants : DWORD
 {
     READYTORUN_PInvokeTransitionFrameSizeInPointerUnits = 11,
-    READYTORUN_ReversePInvokeTransitionFrameSizeInPointerUnits = 2
+    READYTORUN_ReversePInvokeTransitionFrameSizeInPointerUnits = 5
```

Comment:
Can we make this size bump for Windows x86 only?

Reply:
Sure! I wasn't sure if we had a preference around keeping these constants the same between all platforms.